### PR TITLE
Add target_yaw_offset

### PR DESCRIPTION
**Pull request by khanghugo**
Just small steps here. I am in the process of using target_yaw_offset for stuffs and see if any changes needed for hlstrafe/BXT.

### CODE REVIEW

**Comment by khanghugo** [comment]
Hmmm, the name "target_yaw_offset" might seem a bit misleading. What do you think of just "velocity_lock_offset"?

`target_yaw velocity_lock`, ergo `target_yaw velocity_lock_offset`, ergo `change velocity_lock_offset to x over y s`

**Comment by YaLTeR** [comment]
huh, why did the CI not run

**Comment by YaLTeR** [comment]
oh it's because it was disabled for inactivity

**Comment by YaLTeR** [comment]
force push pls so it runs

**Comment by khanghugo** [comment]
push = forced

File: src/read/line.rs [hunk 1]
```diff
@@ -601,6 +601,9 @@ fn line_change(i: &str) -> IResult<Change> {
     let (value, target) = cut(alt((
         map(tag("yaw"), |_| ChangeTarget::Yaw),
         map(tag("pitch"), |_| ChangeTarget::Pitch),
+        map(tag("target_yaw_offset"), |_| {
```
Comment (YaLTeR) [inline src/read/line.rs:604]: Since you're adding new syntax, you should add tests. Check what I did in past commits.

Comment (khanghugo) [reply src/read/line.rs:604]: ~~Hmmm, there are no tests for other `line_change` call. Not sure if I should follow that as well.~~ hmm found it, probably.

**Comment by YaLTeR** [comment]
Thanks

**Pull request merged**